done/static/css/done.css

```diff
@@ -94,4 +94,34 @@
 
 .done_nogrow {
     flex-grow: 0;
-}
\ No newline at end of file
+}
+
+/* Right to left (rtl) language css */
+
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-inner {
+    margin-right: 0;
+}
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-switch {
+    right: 0px;
+}
+
+body.rtl .done_onoffswitch-inner {
+    margin-right: -100%;
+}
+
+body.rtl .done_onoffswitch-inner:before {
+    content: "Unmark";
+    padding-right: 10px;
+    background-color: #EEEEEE;
+    color: #999999;
+    text-align: right;
+}
+
+body.rtl .done_onoffswitch-inner:after {
+    content: "Mark as complete";
```

Comment:
One thing that I want to highlight, There are pre-exist css/text in this file (you can look into done.css) that contain the 'text strings'. I just bind my css with `body.rtl` here. 
The comment same for other places.